roles/openshift_hosted/defaults/main.yml

```diff
@@ -55,6 +55,7 @@ openshift_hosted_routers:
   selector: "{{ openshift_hosted_router_selector | default(None) }}"
   images: "{{ openshift_hosted_router_registryurl }}"
   edits: "{{ openshift_hosted_router_edits }}"
+  extended_validation: "{{ openshift_hosted_router_extended_validation | default(True) | bool }}"
```

Comment:
/cc @sdodson can you please verify this is the right place to add a new default?

Reply:
Yeah this is fine as long as we don't intend to use this outside of openshift_hosted. If it were then move it to openshift_facts.